test/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ target_compile_definitions(ArborX_DetailsAlgorithms.exe PRIVATE BOOST_TEST_MAIN
 target_include_directories(ArborX_DetailsAlgorithms.exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 add_test(NAME ArborX_DetailsAlgorithms_Test COMMAND ./ArborX_DetailsAlgorithms.exe)
 
-add_executable(ArborX_LinearBVH.exe tstLinearBVH.cpp utf_main.cpp)
+add_executable(ArborX_LinearBVH.exe tstLinearBVH.cpp tstKokkosToolsAnnotations.cpp utf_main.cpp)
```

Comment:
Why is it a part of the same executable, and not a separate one?

Reply:
Separate compile unit, same executable.  Not necessary the best place but I think it is fine for now.